packages/contracts/contracts/foundry/Sphinx.sol

```diff
@@ -96,20 +95,25 @@ abstract contract Sphinx {
         vm.makePersistent(address(sphinxUtils));
     }
 
-    function sphinxCollectProposal(string memory _deploymentInfoPath) external {
+    function sphinxCollectProposal(
+        string memory _deploymentInfoPath,
+        string memory _systemContractsFilePath
```

Comment:
The main update to the collection functions is that they contain the Merkle leaf gas estimation code, which requires that the Sphinx system contracts and the user's Gnosis Safe are deployed. (Explanation in the docs below)